pt-BR/mail/chrome/messenger/preferences/receipts.dtd

```diff
@@ -1,5 +1,5 @@
 <!ENTITY dialog.title                  "Confirmações de leitura">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY dialog.title                  "Confirmações de entrega">